lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+
+function getFileName(filePath) {
+  const scriptFileNameRegEx = /([a-zA-Z0-9_\.\-\(\):])+(\.js)/ig;
```

Comment:
This regex is complex, let's ensure it or `getFileName` are thoroughly unit tested.

Reply:
I have covered different Filenames in the test for updating manifest from index.html. `update-manifest-from-html-test.js`

Let me see I missed anything there.

Reply:
I covered following FileName scenarios in my testing
```
 <script src="%PLACEHOLDER[c.js]%"></script>
  <script src="/some_context_path/e.js"></script>
  <script src="chunk-12.js"></script>
  <script src="chunk-12.test.js"></script>
```

Reply:
Also, could you briefly explain what the purpose of the regexp as well?